lifecycle/manager.go

```diff
@@ -54,6 +54,12 @@ func (lcs *LifecycleServicesManager) CreateReleaseBundleFromBundles(rbDetails li
 	return rbService.CreateFromBundles(rbDetails, queryParams, signingKeyName, sourceReleaseBundles)
 }
 
+func (lcs *LifecycleServicesManager) GetReleaseBundleSpecification(rbDetails lifecycle.ReleaseBundleDetails) (lifecycle.ReleaseBundleSpecResponse, error) {
+	rbService := lifecycle.NewReleaseBundlesService(lcs.config.GetServiceDetails(), lcs.client)
+	bs, _, err := rbService.GetReleaseBundleSpec(rbDetails)
+	return bs, err
```

Comment:
```suggestion
	return rbService.GetReleaseBundleSpec(rbDetails)
```
I suggest to remove the unused `body` output from the function if it isn't used.